### PR TITLE
Add CM/ECF research page

### PR DESCRIPTION
**Pull request by cmaczo**
- Adding an “Electronic Filing and Case Management Research” page, which contains:
    - Links to relevant blog posts
    - Links to FOIA’ed documents
- Linking that page from the footer

### CODE REVIEW

File: posts/projects/filing-and-case-management.mdx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+You can keep up with this project on our blog:
+
+- [Report on our interviews with advocates and vendors][advocate-report]
+- [Project launch announcement][launch-blog]
+
+[advocate-report]: /2023/12/12/open-efiling-launch/
+[launch-blog]: /2024/04/29/court-systems-research/
+
+## Existing state contracts for related systems
```
Comment (mlissner) [inline posts/projects/filing-and-case-management.mdx:18]: Per my larger comment, I think the easiest thing is to just remove this part.

File: components/footers.js [hunk 2]
```diff
@@ -87,6 +87,7 @@ export function Footer({ recentPost }) {
             </FooterLink>
             <FooterLink href="/pacer-facts/">PACER Fact Sheet</FooterLink>
             <FooterLink href="/idb-facts/">Integrated Database Fact Sheet</FooterLink>
+            <FooterLink href="/projects/filing-and-case-management/">Filing and Case Management System Research</FooterLink>
```
Comment (mlissner) [inline components/footers.js:90]: Could we get away with this, if we remove the blog links:

```suggestion
            <FooterLink href="/projects/filing-and-case-management/">Court Contracts</FooterLink>
```

**Review by mlissner — CHANGES_REQUESTED** [review]
I think it'd be a bit better if this is just a resources page, with less explanation, sort of like this one: 

https://free.law/non-profit-documents

We could then do a brief blog post (like two paragraphs, max three)  saying what we're up to and linking to it, and we could just post the latest contracts as we get them.

What do you think?

**Comment by mlissner** [comment]
Will you let me know when this is ready for another review?

**Comment by cmaczo** [comment]
@mlissner I think it's ready now.

**Review by mlissner — APPROVED** [review]

**Comment by mlissner** [comment]
Cool. Will you have time to do a short post about this saying:

 - What we did
 - Why we did it
 - What we've got so far
 - What we'll do in the future as we get things

?

**Comment by cmaczo** [comment]
Yes, I will try!

**Pull request merged**